docs/modules/indexes/retrievers/examples/tf_idf.ipynb

```diff
@@ -57,6 +59,29 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c016b266",
+   "metadata": {},
+   "source": [
+    "# Create a New Retriever with Documents\n",
```

Comment:
Can it be only one Title/H1/<single #> on the page? Otherwise, we got weird ToC renderings.
Otherwise, LGTM